substrate/utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -116,6 +116,10 @@ pub struct StorageParams {
 	/// Include child trees in benchmark.
 	#[arg(long)]
 	pub include_child_trees: bool,
+
+	/// Access only a percentage of KV pairs.
+	#[arg(long, value_parser = clap::value_parser!(u8).range(1..=100), default_value_t = 100)]
```

Comment:
Please add this to the test in `benchmark_storage_works.rs` to check that the argument can be parsed.